singolo/js/script.js

```diff
@@ -0,0 +1,253 @@
+const navigation = document.querySelector(".navigation");
+const header__navigation = document.querySelector(".header__navigation");
+const header_content = document.querySelector(".header-content");
+const burger = document.querySelector(".burger-menu");
+const body = document.querySelector("body");
+const bod = document.querySelector(".services");
```

Comment:
You have _body_ and _bod_ varibles. Extremely similar naming can confuse developers which will work with your code in the future.
And below, be careful with naming variables

Reply:
fixed